tests/cpu/UnitTestUtils.h

```diff
@@ -93,6 +93,17 @@ inline bool EqualWithSafeRelError(T value,
         / div) <= eps;
 }
 
+bool StringFloatVecClose(std::string value, std::string expected, float eps);
+
+#define OCIO_CHECK_STR_FLOAT_VEC_CLOSE_FROM(x,y,tol,line)                    \
```

Comment:
These are great, but please move OCIO_CHECK_STR_FLOAT_VEC_CLOSE_FROM and OCIO_CHECK_STR_FLOAT_VEC_CLOSE into tests/testutils/UnitTest.h.  That's where all of the other macros like this live and I think it will be much more discoverable for people writing tests in the future.